FinancialTransactionsApi/V1/UseCase/Interfaces/IGetSuspenseAccountUseCase.cs

```diff
@@ -1,14 +1,12 @@
+using System.Threading.Tasks;
 using FinancialTransactionsApi.V1.Boundary.Request;
 using FinancialTransactionsApi.V1.Boundary.Response;
-using FinancialTransactionsApi.V1.Helpers;
-using Hackney.Core.DynamoDb;
-using System.Collections.Generic;
-using System.Threading.Tasks;
+using FinancialTransactionsApi.V1.Helpers.GeneralModels;
 
 namespace FinancialTransactionsApi.V1.UseCase.Interfaces
 {
     public interface IGetSuspenseAccountUseCase
     {
-        public Task<ResponseWrapper<IEnumerable<TransactionResponse>>> ExecuteAsync(SuspenseAccountQuery query);
+        public Task<PaginatedResponse<TransactionResponse>> ExecuteAsync(SuspenseAccountQuery query);
```

Comment:
Previously method returned `Task<ResponseWrapper<IEnumerable<TransactionResponse>>`
Now it returns `Task<PaginatedResponse<TransactionResponse>>`

Does `PaginatedResponse<TransactionResponse>` contain the `IEnumerable<TransactionResponse>`?

Reply:
Done